src/whylogs/app/session.py

```diff
@@ -150,7 +152,7 @@ def log_dataframe(
             dataset_name, dataset_timestamp, session_timestamp, tags, metadata
         )
 
-        ylog.log_dataframe(df)
+        _ = ylog.log_dataframe(df)
```

Comment:
Python noob here. What's the purpose of the `_` here?

Reply:
@andyndang Just a dummy variable.  A convention to show that there is output but it's not used for anything.

Reply:
throw away variable, I removed it. there was confusion with this api call and previous versions  I found in the notebook that expected a profile from `logger.log_dataframe` .